src/test/java/com/example/Restaurant_Management_System_REST_API/service/RestaurantOrderServiceTest.java

```diff
@@ -127,39 +145,35 @@ public void update_ShouldThrowNotFoundInDatabaseException_WhenRestaurantOrderWit
 
         //Act
         //Assert
-        assertThrows(NotFoundInDatabaseException.class, ()-> restaurantOrderService.update(nonExistedId, restaurantOrderDTO));
+        assertThrows(NotFoundInDatabaseException.class, ()-> restaurantOrderService.update(nonExistedId, restaurantOrderRequestDTO));
     }
 
-    //TODO: this below test will need to be updated after merging new branch
     @Test
     public void update_ShouldInteractWithDependenciesCorrectlyAndReturnRestaurantOrderDTO_WhenIdAndRestaurantOrderDTOsGiven()
             throws NotFoundInDatabaseException {
         //Arrange
         Table table = mock(Table.class);
-        MenuRecord mockMenuRecord = mock(MenuRecord.class);
-        List<MenuRecord> menuRecordList = Arrays.asList(mockMenuRecord);
+        String telephoneNumber = "888888888";
```

Comment:
This is called magic number/string which should be avoided. Instead of this phone number being here it should be contant on top, properly named